core/src/main/scala/scalaz/syntax/std/StringOps.scala

```diff
@@ -24,23 +24,23 @@ final class StringOps(private val self: String) extends AnyVal {
 
   // Parsing functions.
 
-  def parseBoolean: Validation[IllegalArgumentException, Boolean] = s.parseBoolean(self)
+  def parseBoolean: Validation[String, Boolean] = s.parseBoolean(self)
 
-  def parseByte: Validation[NumberFormatException, Byte] = s.parseByte(self)
+  def parseByte: Validation[String, Byte] = s.parseByte(self)
 
-  def parseShort: Validation[NumberFormatException, Short] = s.parseShort(self)
+  def parseShort: Validation[String, Short] = s.parseShort(self)
 
-  def parseInt: Validation[NumberFormatException, Int] = s.parseInt(self)
+  def parseInt: Validation[String, Int] = s.parseInt(self)
 
-  def parseLong: Validation[NumberFormatException, Long] = s.parseLong(self)
+  def parseLong: Validation[String, Long] = s.parseLong(self)
 
-  def parseFloat: Validation[NumberFormatException, Float] = s.parseFloat(self)
+  def parseFloat: Validation[String, Float] = s.parseFloat(self)
 
-  def parseDouble: Validation[NumberFormatException, Double] = s.parseDouble(self)
+  def parseDouble: Validation[String, Double] = s.parseDouble(self)
 
-  def parseBigInt: Validation[NumberFormatException, BigInt] = s.parseBigInt(self)
+  def parseBigInt: Validation[String, BigInt] = s.parseBigInt(self)
 
-  def parseBigDecimal: Validation[NumberFormatException, BigDecimal] = s.parseBigDecimal(self)
+  def parseBigDecimal: Validation[String, BigDecimal] = s.parseBigDecimal(self)
```

Comment:
I'm still in favour or removing the `scala.BigDecimal` constructor and replacing with the java variant (noting the scala bug in comments).